arcgis-ios-sdk-samples/Utility network/Perform valve isolation trace/PerformValveIsolationTrace.storyboard

```diff
@@ -52,7 +54,6 @@
                                                     <nil key="highlightedColor"/>
                                                 </label>
                                             </subviews>
-                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
```

Comment:
No need to add clear color to a view explicitly, as the default color for a `UIView` is clearColor.